packages/manager/apps/ips/src/data/hooks/ip/useDeleteVirtualMac.tsx

```diff
@@ -0,0 +1,48 @@
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+import { ApiError, ApiResponse } from '@ovh-ux/manager-core-api';
+import { useNotifications } from '@ovh-ux/manager-react-components';
+import { deleteVirtualMac, getdedicatedServerVmacQueryKey } from '@/data/api';
+
+export type UseDeleteVirtualMacParams = {
+  serviceName: string;
+  macAddresses: string[];
+  ip: string;
+  onError?: (apiError: ApiError) => void;
+  onSuccess?: (result: ApiResponse<void>) => void;
+};
+
+export const useDeleteVirtualMac = ({
```

Comment:
I don't think it's usefull to create a hook everytime we need to do a useMutation
It won't be reused anywhere outside the DeleteVirtualMac modal in my opinion so no need to split the logic inside a reusable hook to be used only once